lib/src/replaced_element.dart

```diff
@@ -82,7 +83,16 @@ class ImageContentElement extends ReplacedElement {
       if (entry.key.call(attributes, element)) {
         final widget = entry.value.call(context, attributes, element);
         if (widget != null) {
-          return widget;
+          return RawGestureDetector(
```

Comment:
Hmmm did we introduce a regression bug where the `onImageTap` listener was never called?

Reply:
I think so, from when you implemented the image renders.